File: azure-functions/java/src/main/java/com/company/sales/api/models/Payment.java
package com.company.sales.api.models;

import java.math.BigDecimal;
import java.util.UUID;

public class Payment {
    public UUID id;
    public UUID customerId;
    public UUID subscriptionId;
    public String timestamp;
    public BigDecimal amount;
    public String emailAddress;
}

File: azure-functions/csharp/Models/Payment.cs
using System;
using Newtonsoft.Json;

namespace Company.Sales.Api.Models
{
    public class Payment
    {
        [JsonProperty("id")]
        public Guid Id { get; set; }

        [JsonProperty("customerId")]
        public Guid CustomerId { get; set; }

        [JsonProperty("subscriptionId")]
        public Guid SubscriptionId { get; set; }

        [JsonProperty("timestamp")]
        public DateTime Timestamp { get; set; }

        [JsonProperty("amount")]
        public decimal Amount { get; set; }

        [JsonProperty("emailAddress")]
        public string EmailAddress { get; set; }
    }
}